README.md

```diff
@@ -75,23 +79,25 @@ If you don't have an already existing rust installation, you can install it usin
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
-### Install WebAssembly target
+### Clone the repo
 
-Hyperbridge's blockchain runtime compiles to wasm which allows it's code to be forklessly upgraded. In order to build hyperbridge we need the wasm toolchain installed.
+Download a local copy of the repo and checkout the latest release tag
 
 ```bash
-rustup update nightly
-rustup target add wasm32-unknown-unknown --toolchain nightly
+export LATEST_HYPERBRIDGE_RELEASE=v0.4.0
+git clone --branch ${LATEST_HYPERBRIDGE_RELEASE} https://github.com/polytope-labs/hyperbridge.git
```

Comment:
It's a tag not a branch actually

Reply:
isn't it the same approach to checkout a tag? for some reason Git uses `--branch` for checking out tags using that one liner. maybe it should be this instead according to this blog https://www.namehero.com/blog/how-to-clone-a-git-tag/
```suggestion
git clone --branch ${LATEST_HYPERBRIDGE_RELEASE} --single-branch https://github.com/polytope-labs/hyperbridge.git
```

i actually prefer the way it was since it's more readable, so could just change it to:
```bash
export LATEST_TAG=v0.4.0
git clone https://github.com/polytope-labs/hyperbridge.git
cd ./hyperbridge
git checkout ${LATEST_TAG}
```

Reply:
Ahh didn't know you could do that, this works too